src/data/Footer.js

```diff
@@ -0,0 +1,33 @@
+import { HiLink } from "react-icons/hi";
+import { IoMdMail } from "react-icons/io";
+import { PiTiktokLogo } from "react-icons/pi";
+import { FaDiscord } from "react-icons/fa";
+import { AiFillInstagram } from "react-icons/ai";
+export const footer = [
+  {
+    // email
+
+    link: "mailto: ",
```

Comment:
i got the email for the club, so lets replace it rn

ucr.hungry4biscuits@gmail.com